src/Microsoft.ML.Data/Evaluators/MulticlassClassificationEvaluator.cs

```diff
@@ -213,6 +215,11 @@ public override IEnumerable<MetricColumn> GetOverallMetricColumns()
                         overallDvBldr.AddColumn(TopKAccuracy, NumberDataViewType.Double, topKAcc.ToArray());
                     overallDvBldr.AddColumn(PerClassLogLoss, aggregator.GetSlotNames, NumberDataViewType.Double, perClassLogLoss.ToArray());
 
+                    ValueGetter<VBuffer<ReadOnlyMemory<char>>> getKSlotNames =
```

Comment:
If we limit the Top-K array, the same as [above](https://github.com/dotnet/machinelearning/pull/5395/files#r514685584) applies here for this column that is added.